### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role.git`](https://github.com/cisagov/skeleton-ansible-role.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-clamav.git ansible-role-clamav
    cd ansible-role-clamav
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md meta/main.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> **Note**
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
LGTM! 👍

File: molecule/default/prepare.yml [hunk 1]
```diff
@@ -13,18 +13,23 @@
...
+    # next line.
+    - name: Wait for systemd to complete initialization # noqa command-instead-of-module
       ansible.builtin.command: systemctl is-system-running
       register: systemctl_status
       until: "'running' in systemctl_status.stdout"
       retries: 30
       delay: 5
-      when: ansible_service_mgr == 'systemd'
+      when: ansible_service_mgr == "systemd"
+      changed_when: false
```
Comment (mcdonnnj) [inline molecule/default/prepare.yml]: Did we want to sort these additional task configuration keys?

File: tasks/setup_manual.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 # This task runs freshclam and controls the virus database creation.
-# It happens automatically with SystemD when registering freshclam service.
+# It happens automatically with SystemD when registering freshclam
```
Comment (mcdonnnj) [inline tasks/setup_manual.yml]: Just a thought to improve readability.

```suggestion
# It happens automatically with SystemD when registering the freshclam
```

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪💪💪 I have a couple of small thoughts for your consideration but otherwise this LGTM ✔

File: molecule/default/prepare.yml (hunk 1 above)
Comment (jsf9k) [reply molecule/default/prepare.yml]: Please see commit d016299.

**Pull request merged**